### ISSUE TITLE
New sessions don't save and can't save manually

### ISSUE THREAD

**Issue by pmsobrado** [sienori/Tab-Session-Manager#1356]
## Short description
I'm gettings failed to save session whenever I tried to save manually. Also, the last automatic sessions saved seems to be from weeks ago, there no recent sessions saved. I tried uninstalling and reinstalling.

## Steps to reproduce

1. Just try to save manually or wait for automatically save to trigger and see how it does not save anything.

## Expected result
Sessions should be save manually and automatically.

## Actual result
Failed to save session red message (I can't take screenshots) and no recent sessions automatically saved.

## Platform information
* Platform (OS): Ubuntu 18.04
* Version of browser: 113.0.2 (for some reason Ubuntu 18.04 has stopped on this version)
* Version of Tab Session Manager: 7.0.1

**Comment by pmsobrado** [comment]
More info: 7.0.0 version also fails, but latest 6.x does not. So I suspect some change does not work well with my FF/Ubuntu, or with the combination of FF and Ubuntu.

**Comment by freemedom** [comment]
yeah, since this updated to Manifest v3, many bugs about compatibility

**Comment by pmsobrado** [comment]
So, any fix?

**Comment by freemedom** [comment]
@pmsobrado i'm not the developer.

**Comment by pmsobrado** [comment]
I know, but maybe the developer sees this thread.
